lerna.json

```diff
@@ -1,6 +1,6 @@
 {
   "packages": ["packages/*", "demo"],
-  "version": "1.0.0-beta2",
+  "version": "1.0.0-beta3",
```

Comment:
I think lerna does this atuomatically as a separate commit

Reply:
I mean, when you run `npx lerna version`

Reply:
Reverted